app/controllers/article_controller.rb

```diff
@@ -94,15 +71,14 @@ def article_category
     else
       @article.categories.delete(@category)
     end
-    render :plain => "success"
+    render plain: 'success'
   end
 
   def combine_duplicate
-    #@article contains "to" article
-    params[:from_article_ids].each do |from_article_id|
-      from_article = Article.find(from_article_id)
-      combine_articles(from_article, @article)
-    end
+    Article::Combine.call(
```

Comment:
I like moving `combine_articles` logic into the model, but am confused why we're doing this call syntax (again).

Reply:
Interactors -- got it!